samples/maps/marker-clusters/europe/demo.js

```diff
@@ -0,0 +1,77 @@
+Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@1e9e659c2d60fbe27ef0b41e2f93112dd68fb7a3/samples/data/european-train-stations-near-airports.json', function (data) {
+    Highcharts.mapChart('container', {
+        chart: {
+            map: 'custom/europe'
+        },
+        title: {
+            text: 'European Train Stations Near Airports'
+        },
+        subtitle: {
+            text: 'Data source: https://github.com/trainline-eu/stations'
+        },
+        mapNavigation: {
+            enabled: true
+        },
+        tooltip: {
+            headerFormat: '',
+            pointFormat: '<b>{point.name}</b><br>Lat: {point.lat}, Lon: {point.lon}'
+        },
+        colorAxis: {
+            min: 0,
+            max: 15
+        },
+        plotOptions: {
+            mappoint: {
+                cluster: {
+                    enabled: true,
+                    allowOverlap: false,
+                    animation: {
+                        duration: 450
+                    },
+                    layoutAlgorithm: {
+                        type: 'grid',
+                        gridSize: 60
+                    },
+                    zones: [{
+                        from: 1,
+                        to: 4,
+                        style: {
```

Comment:
This doesn't take effect, it appears from the source that it should be `zones.marker.radius`.